llvm/include/llvm/IR/IntrinsicsAArch64.td

```diff
@@ -3107,6 +3107,26 @@ let TargetPrefix = "aarch64" in {
     }
   }
 
+  class SME_OuterProduct_TMOP_Intrinsic
+     : DefaultAttrsIntrinsic<[],
+         [llvm_i32_ty,
+          llvm_anyvector_ty,
+          LLVMMatchType<0>,
+          LLVMMatchType<0>,
+          llvm_nxv16i8_ty,
+          llvm_i32_ty],
+         [ImmArg<ArgIndex<0>>, ImmArg<ArgIndex<5>>,
+          IntrInaccessibleMemOnly]>;
+
+  def int_aarch64_sme_za16_ftmopa : SME_OuterProduct_TMOP_Intrinsic;
+  def int_aarch64_sme_za16_bftmopa : SME_OuterProduct_TMOP_Intrinsic;
```

Comment:
Do we need a int_aarch64_sme_za16_bftmopa , should it not be enough the vector size <vscale x 16 x b16>

Reply:
Sure, now adjusted. (My original patch used `int_aarch64_sme_za16_tmopa` where they could be deduced by type. I'll change it back to this. The LLVM IR name won't have 'bf' in the name, and will become plain `@llvm.aarch64.sme.tmopa.za16.nxv8bf16`).